lms/djangoapps/bulk_email/models.py

```diff
@@ -451,6 +451,17 @@ def __unicode__(self):
         return u"Course '{}': Instructor Email {}Enabled".format(text_type(self.course_id), not_en)
 
 
+# .. feature_toggle_name: require_course_email_auth
+# .. feature_toggle_type: ConfigurationModel
+# .. feature_toggle_default: True (enabled)
+# .. feature_toggle_description: If the flag is enabled, course-specific authorization is required, and the course_id is either not provided or not authorixed, the feature is not available.
+# .. feature_toggle_category: bulk email
+# .. feature_toggle_use_cases: open_edx
+# .. feature_toggle_creation_date: 2016-05-05
+# .. feature_toggle_expiration_date: None
+# .. feature_toggle_warnings: None
+# .. feature_toggle_tickets: None
+# .. feature_toggle_status: supported
```

Comment:
Yeah - the `feature_toggle_` prefix seems wordy and takes away from the crux of the content.  We may prefer just using `toggle_` as the prefix.